src/exchange/contractService.ts

```diff
@@ -163,12 +161,41 @@ export class ContractCalls {
     //serialize
     const separator = "||||"; // Choose a separator that won't appear in txBytes or signature
     const combinedData = `${signedTx.bytes}${separator}${signedTx.signature}`;
+    console.log("signedTx.bytes: ", signedTx.bytes);
+    console.log("signedTx.signature: ", signedTx.signature);
     // Encode to hex for transmission
     const encodedData = Buffer.from(combinedData, "utf-8").toString("hex");
 
     return encodedData;
   };
 
+  upsertSubAccountContractCallRawTransaction = async (
+    account: string,
+    status: boolean,
```

Comment:
I beleive we removed status in https://github.com/fireflyprotocol/library-sui/pull/143